ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/execution/PullQueryExecutorTest.java

```diff
@@ -75,8 +74,7 @@ public void shouldThrowExceptionIfConfigDisabled() {
       // When:
       final Exception e = assertThrows(
           KsqlException.class,
-          () -> pullQueryExecutor.execute(query, engine.getServiceContext(), Optional.empty(),
-              Optional.empty())
+          () -> pullQueryExecutor.execute(query, engine.getServiceContext(), Optional.empty(), 0L)
```

Comment:
Would be good to test its passing an appropriate start time here. Can we inject a clock?